src/LaunchDarkly.ClientSdk/PlatformSpecific/DeviceInfo.maui.cs

```diff
@@ -0,0 +1,48 @@
+using LaunchDarkly.Sdk.EnvReporting.LayerModels;
+using Devices = Microsoft.Maui.Devices;
+
+namespace LaunchDarkly.Sdk.Client.PlatformSpecific
+{
+    internal static partial class DeviceInfo
+    {
+        internal static OsInfo? GetOsInfo() =>
+            new OsInfo(
+                PlatformToFamilyString(Devices.DeviceInfo.Current.Platform),
+                Devices.DeviceInfo.Current.Platform.ToString(),
+                Devices.DeviceInfo.Current.VersionString
+            );
+
+        internal static LaunchDarkly.Sdk.EnvReporting.LayerModels.DeviceInfo? GetDeviceInfo() =>
+            new EnvReporting.LayerModels.DeviceInfo(
+                Devices.DeviceInfo.Current.Manufacturer,
+                Devices.DeviceInfo.Current.Model
+            );
+
+        private static string PlatformToFamilyString(Devices.DevicePlatform platform) {
+            if (platform == Devices.DevicePlatform.Android)
+            {
+                return "Android";
+            }
+            else if (platform == Devices.DevicePlatform.iOS ||
+                platform == Devices.DevicePlatform.watchOS ||
+                platform == Devices.DevicePlatform.tvOS ||
+                platform == Devices.DevicePlatform.macOS ||
+                platform == Devices.DevicePlatform.MacCatalyst)
+            {
+                return "Apple";
+            }
+            else if(platform == Devices.DevicePlatform.UWP || platform == Devices.DevicePlatform.WinUI)
```

Comment:
For reviewers: While UWP is obsolete, it is better to fully map the enum than remove the obsolete value.

Reply:
These should probably have comment suppression on each one to make that clear. Though I don't know if the reasoning is accurate. 

First being it doesn't seem to be an enum, but a class with static instances. Second they tell you not to use it, so probably for reason. It feels to me that adding it just means that our code will break when they remove the instance, but it doesn't add any value before then.

Reply:
Removed.